src/main/java/uk/gov/hmcts/darts/audio/exception/FileGenerationError.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.darts.audio.exception;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.darts.common.exception.DartsApiError;
+
+@Getter
+@RequiredArgsConstructor
+public enum FileGenerationError implements DartsApiError {
```

Comment:
I think we only need one enum per feature, so it would be better to add a `FAILED_TO_GENERATE_README_FILE` value into the existing `AudioError` enum, and delete `FileGenerationError`.

Reply:
We spoke about this and decided that as its going to KEDA it makes sense to use the same audio error message